authors/apps/authentication/tests/test_password_request.py

```diff
@@ -0,0 +1,116 @@
+from rest_framework import status
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class TestResetPassword(TestCase):
+    """Test data"""
+
+    def setUp(self):
+        client = Client()
+        self.register_url = reverse("auth:register")
+
+    # Data with valid user credentials
+        self.valid_user_credentials = {
+            "user": {
+                "username": "Alpha",
+                "email": "alphaandela@gmail.com",
+                "password": "@Alpha254"
+            }
+        }
+        self.user_email = [
+            {
+                "user": {
+                    "email": ""
+                }
+            },
+            {
+                "user": {
+                    "email": "newuser@gmail.com"
+                }
+            },
+            {
+                "user": {
+                    "email": "alpha@andela"
+                }
+            },
+            {
+                "user": {
+                    "email": "alphaandela@gmail.com"
+                }
+            },
+
+        ]
+
+    def user_registration(self):
+        """Register a User"""
+        response = self.client.post(
+            self.register_url,
+            self.valid_user_credentials,
+            content_type='application/json'
+        )
+        return response
+
+    def get_email(self):
+        """Get email of the registered user"""
+        email = self.user_registration().data['email']
+        mail = {
+            "user": {
+                "email": email
+            }
+        }
+        return mail
+
+    def submit_email(self, data):
+        return self.client.post(
+            reverse("auth:reset_request"),
+            data,
+            content_type='application/json')
+
+    def test_request_reset_without_email(self):
+        """Test request password reset without an email"""
+        response = self.submit_email(self.user_email[0])
+
+        self.assertEqual(
+            response.status_code,
+            status.HTTP_400_BAD_REQUEST)
+
+        self.assertEqual(
+            response.json(),
+            {"message": 'Please provide an email'})
```

Comment:
@BoltC0rt3z Nice work on testing the error messages but you can import the messages file to assert the response messages in your tests